src/pages/household/output/HouseholdOutput.jsx

```diff
@@ -37,12 +38,26 @@ export default function HouseholdOutput(props) {
     baselineLabel = "Current law";
   }
   let policyLabel;
-  if (reformLabel !== "Current law" && baselineLabel === "Current law") {
+  if (reformLabel === "Current law" && baselineLabel === "Current law") {
+    policyLabel = "Current law";
+  } else if (reformLabel !== "Current law" && baselineLabel === "Current law") {
     policyLabel = reformLabel;
   } else {
     policyLabel = `${reformLabel} (compared against ${baselineLabel})`;
   }
   let pane;
+  let title;
+  if (policyLabel) {
+    if (focus === "householdOutput.netIncome") {
```

Comment:
Previously, each sub-component within the code managed its own title. Now that we've enabled helmet, can you return that logic so that each component can be more self-contained and we avoid this nested if?